### PR TITLE
Watch3d separate

### PR DESCRIPTION
**Pull request by aosyatnik**
### Purpose

Link to YouTrack:
[MAGN-8962](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-8962) Enable publish Watch3D node with Customizer

I created 2 projects:
* Watch3DNodeModels
* Watch3DNodeModelsWpf

RUN Self CI, no fallen tests, except excel tests.
### Declarations

- [x] The code base is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [x] All tests pass using the self-service CI.

### Reviewers

@pboyer 

### FYIs

@marimano 
@pbidenko

### CODE REVIEW

**Comment by Racel** [comment]
@pboyer - PTAL.

File: src/Dynamo.Mono.sln [hunk 1]
```diff
@@ -1,8 +1,6 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.23107.0
-MinimumVisualStudioVersion = 10.0.40219.1
+# Visual Studio 2012
```
Comment (pboyer) [inline src/Dynamo.Mono.sln]: Let's leave this unchanged.

File: src/Dynamo.All.sln [hunk 2]
```diff
@@ -1,8 +1,6 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.23107.0
-MinimumVisualStudioVersion = 10.0.40219.1
+# Visual Studio 2012
```
Comment (pboyer) [inline src/Dynamo.All.sln]: Let's leave this unchanged.

**Comment by pboyer** [comment]
@aosyatnik This looks awesome. If you can fix the above lines, that would be great. After that, LGTM and feel free to merge.

**Pull request merged**